core/lib/commitment_utils/src/lib.rs

```diff
@@ -1,23 +1,33 @@
 //! Utils for commitment calculation.
+use multivm::utils::get_used_bootloader_memory_bytes;
 use zkevm_test_harness::witness::utils::{
     events_queue_commitment_fixed, initial_heap_content_commitment_fixed,
```

Comment:
double-checking: there is no difference in commitment generation between 1.4.0 and 1.4.1, right?

Reply:
to the best of my knowledge no, but I'll probably just use the newer methods just in case

Reply:
Making it the other way would create a big diff for this PR, so I'll double check with the crypto team + create a follow up PR